modules/mobile-agents/android/client/iDPProxy/src/main/java/org/wso2/emm/agent/proxy/AccessTokenHandler.java

```diff
@@ -143,8 +139,7 @@ protected void onPostExecute(String result) {
                         editor.putString(Constants.DATE_LABEL, strDate);
                         editor.commit();
 
-                        identityProxy.receiveAccessToken(responseCode, Constants.SUCCESS_RESPONSE,
-                                token);
+                        identityProxy.receiveAccessToken(responseCode, Constants.SUCCESS_RESPONSE, token);
                     } catch (JSONException e) {
                         Log.e(TAG, "Invalid JSON format." + e);
```

Comment:
We don't seem to log the exception trace in this particular line. Again, even though this doesn't appear to be something that you'd added, we'd have to fix it.